base/array.jl

```diff
@@ -2,6 +2,38 @@
 
 ## array.jl: Dense arrays
 
+"""
+    Array{T}(dims)
+    Vector{T}(m)
+    Matrix{T}(m, n)
+
+`Array{T}(dims)` constructs an uninitialized dense array with element type `T`. `dims` may
+be a tuple or a series of integer arguments, each giving the size of the corresponding
+dimension. `Vector{T}(m)` and `Matrix{T}(m, n)` are stricly equivalent aliases to create one-
+and two-dimensional arrays.
+
+The syntax `Array(T, dims)` is also available, but deprecated.
+
+
```

Comment:
OK. Yet, running `"a" Array" at the REPL adds "a" to the existing docstring.

Indeed I would like to attach the message to a more specific signature, but these are defined as methods for `convert`, and users are more likely to look for them under `Array`. If every type adds docstrings to `convert`, its help page is going to be unusable pretty soon.

That `convert` fallback stuff might be more confusing than useful...